tests/test_htmIndex.py

```diff
@@ -475,6 +480,36 @@ def testProperMotion(self):
         self.assertFloatsAlmostEqual(references["coord_raErr"], predictedRaErr)
         self.assertFloatsAlmostEqual(references["coord_decErr"], predictedDecErr)
 
+    def testLoadVersion0(self):
+        """Test reading a pre-written format_version=0 (Jy flux) catalog.
+        It should be converted to have nJy fluxes.
+        """
+        path = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'data/version0')
+        loader = LoadIndexedReferenceObjectsTask(butler=dafPersist.Butler(path))
+        self.assertEqual(loader.dataset_config.format_version, 0)
+        result = loader.loadSkyCircle(make_coord(10, 20), 5*lsst.geom.degrees, filterName='a')
+        # TODO: assert that log.warn messages are emitted.
```

Comment:
Is this TODO associated with a ticket?

Reply:
I don't know. It was a todo for me while I was writing it, but I didn't actually implement that part, and I'm not sure how much it matters. Is it worth checking that we emit some kind of message for old catalogs?

Reply:
There are a lot of place we don't check log messages.  If you are not going to do it, I'm fine just removing the TODOs.